opm/simulators/flow/GenericOutputBlackoilModule.cpp

```diff
@@ -263,57 +263,6 @@ calc_inplace(std::map<std::string, double>& miscSummaryData,
     return inplace;
 }
 
-template<class FluidSystem>
-void GenericOutputBlackoilModule<FluidSystem>::
-outputFipAndResvLog(const Inplace& inplace,
```

Comment:
this function assume that `this->initialInplace()` always have value, so it is moved to outputBlackoilModule.hpp.